buildconfig/Jenkins/Conda/download-and-install-mambaforge

```diff
@@ -11,11 +11,19 @@ EXPECTED_MAMBAFORGE_PATH=$1
 EXPECTED_CONDA_PATH=$2
 CLEAN_BUILD=$3
 
-MAMBAFORGE_SCRIPT_NAME=Mambaforge-$(uname)-$(uname -m).sh
+# Only supporting x86 at this time for the build process
```

Comment:
Nit: this should be `AMD64`, or `X86_64`.
Otherwise it reads as we're a 32-bit build